src/main/java/com/synopsys/integration/alert/common/ConfigurationFieldModelConverter.java

```diff
@@ -62,15 +63,23 @@ public final FieldAccessor convertToFieldAccessor(final FieldModel fieldModel) t
         return convertToConfigurationFieldModelMap(fieldModel);
     }
 
-    public final Optional<ConfigurationFieldModel> convertFromDefinedFieldModel(final DefinedFieldModel definedFieldModel, final String value) {
+    public final Optional<ConfigurationFieldModel> convertFromDefinedFieldModel(final DefinedFieldModel definedFieldModel, final String value, final boolean isSet) {
         final Optional<ConfigurationFieldModel> configurationModel = createEmptyModel(definedFieldModel);
-        configurationModel.ifPresent(model -> model.setFieldValue(value));
+        final boolean actualSetValue = isSet || StringUtils.isNotBlank(value);
+        configurationModel.ifPresent(model -> {
```

Comment:
We could abstract the ifPresent functionality into a method to reuse here and bellow.

Reply:
I dont think we would get value from that since the body of this ifPresent uses different method calls than the one below.